### ISSUE TITLE
T430 + i7-3720QM power supply Problems!

### ISSUE THREAD

**Issue by schwarzesloch** [n4ru/1vyrain#14]
Hello,

i have used the jailbreak on my T430 with an i7-3720QM With the result that I already have the problem with two mainbaords that the system thinks that the used power supply does not provide enough power.
No matter if with a 90W, 135W or 170W.
None is accepted.
65W I have not tested.

The T430 has no Nvidia, I didn't change anything on the CPU power limits, just disabled Intel ME.

If I deactivate MCore Multi-Processing in the Bios, the power supply is recognized and can be used.

The T430 can be started without battery with the power supply, but as soon as the T430 has a battery, the T430 goes to battery operation.

Bios-downgrade to 2.64 doesn't bring any improvement

Translated from german to english with deepl.com

Translated with www.DeepL.com/Translator (free version)

**Comment by schwarzesloch** [comment]
Hello,

I have just installed Win 10 1709 on another SSD
There I installed the Thinkpad Power Manager with all drivers from Lenovo System Update.
I didn't install anything via the Windows Update search.

When I start Win, I get a message that the used power supply does not have enough power (in my case 135W) but the battery is charging and the Power Manager reports that everything is ok.

Now when I boot Win 10 1909 from the other SSD, the T430 only runs on battery.

Since I also use Linux on the T430, there's something else:
Linux uses the settings that were last active.
When I start Win 1709, then Linux, the power supply is used normally, when I start Win 1909, then Linux runs Linux only with the battery and not with the power supply.

Translated with www.DeepL.com/Translator (free version)
